packages/client-app/src/views/Request/components/OAuth2.vue

```diff
@@ -58,36 +58,57 @@ const handleAuthorize = async () => {
   </DataTableRow>
 
   <template v-else>
-    <DataTableRow v-if="schemeModel.flowKey !== 'clientCredentials'">
+    <DataTableRow
+      v-if="['implicit', 'authorizationCode'].includes(schemeModel.flowKey)">
       <!-- Redirect URI -->
       <RequestAuthDataTableInput
         id="oauth2-redirect-uri"
         :modelValue="activeScheme.scheme.redirectUri"
         placeholder="https://galaxy.scalar.com/callback"
-        @update:modelValue="(v) => props.updateScheme('redirectUri', v)">
+        @update:modelValue="(v) => updateScheme('redirectUri', v)">
         Redirect URI
       </RequestAuthDataTableInput>
     </DataTableRow>
 
+    <!-- Username and password -->
+    <template
+      v-if="schemeModel.flowKey === 'password' && 'value' in activeScheme.flow">
```

Comment:
for some reason our mutators blow up on `flow.password.password` so I just went with value and secondValue again